### PR TITLE
Add test data for Convolution3D layer

### PR DESCRIPTION
**Pull request by l-bat**

### CODE REVIEW

**Comment by alalek** [comment]
Don't re-open PRs. There are almost no problems which can't be fixed without re-opening PR.
CI doesn't look on PRs from `opencv_extra` at all. Only branches are taken into account for testing.
Builds are triggered by pushing commits into the branch of the **main** PR only.

opencv_extra's PR is necessary for reviewing and merging.

---

> Merge branch '3.4'

These commits should not be here. You need to rebase your branch.

File: testdata/dnn/onnx/generate_onnx_models.py [hunk 1]
```diff
@@ -263,3 +263,11 @@ def forward(self, x):
 model = Unsqueeze()
 model.eval()
 save_data_and_model("unsqueeze", input, model)
+
+input = Variable(torch.randn(1, 3, 8, 12, 16))
```
Comment (dkurt) [inline testdata/dnn/onnx/generate_onnx_models.py]: Maybe we can do it smaller?

```
1*3*8*12*16*4 ~ 18.5KB
```
but, in exampe
```
input = Variable(torch.randn(1, 2, 3, 4, 5))
1*2*3*4*5*4 ~ 0.48KB (x38 smaller but also has vary dimensions).
```

Comment (l-bat) [reply testdata/dnn/onnx/generate_onnx_models.py]: done

File: testdata/dnn/tensorflow/generate_tf_models.py [hunk 2]
```diff
@@ -704,6 +707,10 @@ def pad_depth(x, desired_channels):
 
 ################################################################################
 
+inp = tf.placeholder(tf.float32, [1, 8, 12, 16, 3], 'input') # NDHWC format
```
Comment (dkurt) [inline testdata/dnn/tensorflow/generate_tf_models.py]: Can we reduce test data here as well?

**Comment by dkurt** [comment]
@l-bat, please resolve merge conflicts.

**Pull request merged**